configuration/pipeline_config.json

```
@@ -3,6 +3,7 @@
   "RawDataSources": [
     {
       "SourceType": "RapidPro",
+      "SourceName": "Jambo",
```

```suggestion
      "SourceName": "jambo",
```To match the stringvalue

---

Jambo matches the match_value, which is what it's supposed to equal